suclass/lms/templates/header.html

```diff
@@ -12,18 +12,16 @@
   <img src="${static.url('images/stanford-S-logo.png')}" height="35" alt="Home Page" />
 </%block>
 
-% if course:
-    % if UserProfile.has_registered(user):
-        <%block name="navigation_global_links_authenticated">
-            %if settings.FEATURES.get('ENABLE_SYSADMIN_DASHBOARD','') and user.is_staff:
-                <li class="item">
-                    ## Translators: This is short for "System administration".
-                    <a href="${reverse('sysadmin')}">${_("Sysadmin")}</a>
-                </li>
-            %endif
-        </%block>
-    %endif
-%else:
-    <%block name="navigation_global_links"></%block>
-    <%block name="navigation_other_global_links"></%block>
+% if UserProfile.has_registered(user):
```

Comment:
Let's try unforking all of these blocks too.
Most links are already conditional based on feature flags.